keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/artifacts/BuildMetadata.kt

```diff
@@ -13,7 +17,23 @@ data class BuildMetadata(
   val startedTs: String? = null,
   val completedTs: String? = null,
   val status: String? = null
-)
+) {
+  fun getStartedAtInstant(): Instant? =
```

Comment:
Rather than having this as a method I think the Kotlin-y way to do it would be:

```
val startedAtInstant: Instant?
  get() = // .. the rest of your implementation